src/core/query-builder.utils.ts

```diff
@@ -1,17 +1,28 @@
 import { QueryBuilderCustomOperation } from "smart-webcomponents-react";
```

Comment:
Maybe not in this PR but it might make sense to think of creating a custom hook out of this file

Reply:
Taking into consideration in future work.